classes/State.php

```diff
@@ -409,4 +391,9 @@ public function setProcessTimestamp(string $processTimestamp): void
     {
         $this->processTimestamp = $processTimestamp;
     }
+
+    public function isInitialized(): bool
```

Comment:
Can we have a description on this method or the variable to understand the meaning of this new property?